packages/wdio-cli/src/utils.ts

```diff
@@ -590,16 +590,20 @@ export async function getAnswers(yes: boolean): Promise<Questionnair> {
 }
 
 export function getPathForFileGeneration(answers: Questionnair, projectRootDir: string) {
+    const specAnswer = answers.specs || ''
+    const stepDefinitionAnswer = answers.stepDefinitions || ''
+    const pageObjectAnswer = answers.pages || ''
+
     const destSpecRootPath = path.resolve(
         projectRootDir,
-        path.dirname(answers.specs || '').replace(/\*\*$/, ''))
+        (path.dirname(specAnswer) === '.' ? path.resolve(specAnswer) : path.dirname(specAnswer)).replace(/\*\*$/, ''))
 
-    const destStepRootPath = path.resolve(projectRootDir, path.dirname(answers.stepDefinitions || ''))
+    const destStepRootPath = path.resolve(projectRootDir, (path.dirname(stepDefinitionAnswer) === '.' ? path.resolve(stepDefinitionAnswer) : path.dirname(stepDefinitionAnswer)))
 
     const destPageObjectRootPath = answers.usePageObjects
         ? path.resolve(
             projectRootDir,
-            path.dirname(answers.pages || '').replace(/\*\*$/, ''))
+            (path.dirname(pageObjectAnswer) === '.' ? path.resolve(pageObjectAnswer) : path.dirname(pageObjectAnswer)).replace(/\*\*$/, ''))
```

Comment:
Let's make a function that automatically resolves this by taking the answer, check if . and resolve if needed